test/Spec.hs

```diff
@@ -9,10 +9,9 @@ import Control.Monad
 import Data.Text.Lazy (Text)
 import qualified Data.Text.Lazy.IO as TIO
 
-compileFile :: FilePath -> IO (Handle, Handle, ProcessHandle)
-compileFile f = do
-  -- TODO: Add tex file examples!!
-  (_, Just hout, Just herr, ph) <- createProcess (proc "stack" ["exec", "Naproche-SAD", "--", f, "-t", "25"])
+compileFile :: [String] -> FilePath -> IO (Handle, Handle, ProcessHandle)
+compileFile flags f = do
+  (_, Just hout, Just herr, ph) <- createProcess (proc "stack" (["exec", "Naproche-SAD", "--", f] ++ flags))
```

Comment:
Did you run into trouble with `-t 25`? If I had to guess it might be useful for CI.

Reply:
30 was just an arbitrary number I came up with because I was scared that `chinese.ftl` wouldn't run in time. Now that `chinese.ftl` is not in the testsuite anymore, I will revert this back to `-t 25`.

Reply:
I reverted it back to `-t 25`.